orga/app/components/routes/authenticated/campaigns/details/analysis-tab.js

```diff
@@ -13,7 +13,9 @@ export default class AnalysisTab extends Component {
 
   @action
   sortRecommendationOrder(order) {
-    if (order === 'desc') {
+    if (!this.sortedRecommendations) {
```

Comment:
C'est possible d'être dans ce cas ? On a pas un tableau vide quand il n'y a pas de recommendations  ?

Reply:
la levée d'une erreur nous montrait bien qu'on tombait dans ce cas... :/ on avait un `cannot read property sort of undefined` truc dans le genre